app/src/main/AndroidManifest.xml

```diff
@@ -49,6 +49,12 @@
             android:enabled="true" />
 
         <receiver android:name=".Receivers.NotificationAlarmReceiver" />
+        <receiver android:name=".Receivers.NetworkDialogHandler">
```

Comment:
No need for the receiver to be declared here if you are registering it in the `onCreate` of the Application.